nireports/interfaces/mosaic.py

```diff
@@ -112,6 +112,29 @@ def _run_interface(self, runtime):
 class _PlotMosaicInputSpec(_PlotBaseInputSpec):
     bbox_mask_file = File(exists=True, desc="brain mask")
     only_noise = traits.Bool(False, desc="plot only noise")
+    main_view = traits.Enum(
+        "axial",
+        "sagittal",
+        "coronal",
+        default="axial",
+        usedefault=True,
+    )
+    addon_view1 = traits.Enum(
+        "sagittal",
+        "axial",
+        "coronal",
+        None,
+        default="sagittal",
+        usedefault=True,
+    )
+    addon_view2 = traits.Enum(
+        None,
+        "axial",
+        "sagittal",
+        "coronal",
+        default=None,
+        usedefault=True,
+    )
```

Comment:
Would it make sense to make this
```suggestion
    view = traits.List(
        traits.Enum("axial", "sagittal", "coronal"),
        default=["axial", "sagittal"],
        help="Sequence of views to plot (up to three)",
        usedefault=True,
    )
```

I think there's minLength/maxLength or something as well.